### ISSUE TITLE
Would this be upstream in jenkins-job-builder?

### ISSUE THREAD

**Issue by ajqlee** [rusty-dev/jenkins-job-builder-pipeline#5]
I got a feedback on my ITP(Intend To Package) ticket on Debian BTS #851542:
    https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=851542

> Message #22 received at 851542@bugs.debian.org (full text, mbox, reply):
> From: Jeremy Stanley <fungi@yuggoth.org>
> To: Andrew Lee <ajqlee@debian.org>, 851542@bugs.debian.org
> Subject: Re: Bug#851542: ITP: jenkins-job-builder-pipelin -- pipeline job generation plugin for jenkins-job-builder
> Date: Mon, 16 Jan 2017 16:03:24 +0000
> [Message part 1 (text/plain, inline)]
> According to the JJB authors, this plugin was never proposed
> upstream but there is another implementation which has been and is
> likely to merge soon: https://review.openstack.org/323744
> 
> That is to say, it's unlikely you're going to want to maintain a
> redundant/conflicting package for a feature which will probably ship
> in the next JJB release anyway.
> -- 
> Jeremy Stanley

What's the difference between this plugin and the one Jeremy mentioned?
Would this be upstream in jenkins-job-builder?

Best regards,
-Andrew

**Comment by rusty-dev** [comment]
Yeah, I never proposed it to upstream in JJB, I just made it for personal use and haven't had the time\investment to do a proper proposal.
I've looked at the other plugin you mentioned, looks very similar. To be honest, there's not much to it, so they are bound to be pretty much the same.
So, I would agree that it's better for you to wait and use the built-in plugin in the future.
